index.rst

```diff
@@ -352,28 +385,145 @@ To make sure the system would work properly we had to enable two additional feat
+Considering that the :ref:`initial configuration <Appendix-Initial-Shared-Memory-OpenSplice-Configuration>` for the daemons allocated 10 times as much shared memory space then the `standard <https://istkb.adlinktech.com/article/vortex-opensplice-dds-configuration/>`__, we also increased by 10 times the amount of memory available for other daemon services.
+
+In addition to increasing resources for the daemons we also enabled two new features that helps reduce system load; squash participants and lazy alignment.
+
+The squash participant feature allow daemons to "merge" the data structure of all the applications attached to it into a single data structure.
+Therefore, instead of presenting several "participants" to the network, the daemon presents a single unified system that is much leaner to maintain.
+This feature no only helps reduce the data structure complexity but is also reduces discovery time for new components in the system.
+The drawback to squashing participants is that we loose the ability to inspect individual applications data structure.
+Since this is a DDS capability that was not being used in the first place, it seems like a good compromise to adopt this feature.
+
+By default, when an application creates a DDS domain participant and join a partition namespace, DDS allocates resources and start making data on the partition available to it right away.
+This happens even if the application does not create a data reader for a topic in that partition.
+Although this feature helps reduce subscription times and also increase data recovery, it comes at the expense of resources from the application or the daemon it is attached to.
+In general, we are not relying on DDS history capability to store component data in case of a system crash, which means we have limited use for the default behavior.
+In order to allow for a leaner daemon the configuration was changed to use "lazy alignment".
+With this feature, the readers will only receive and store data that is requested by the application or, in case of the daemons, application that are attached to them.
+
+The final configuration we arrived at after extensive testing with the system can be seen in :ref:`Appendix-Final-Shared-Memory-OpenSplice-Configuration`.
+
+.. _Implementing-a-new-DDS-data-partitioning-schema:
+
+Implementing a new DDS data partitioning schema
+-----------------------------------------------
 
-Overall the preliminary results indicate that using the OpenSplice shared memory mode solves the issues we observed previously with a large scale system.
+It is important to note that the :ref:`configuration <Appendix-Final-Shared-Memory-OpenSplice-Configuration>` discussed in :ref:`Increasing-OpenSplice-daemon-resources` affect only the daemons and not those processes that will still rely on single process mode.
```

Comment:
Is "single process mode" the same as "standard mode" (a term you used above)? It would help to adopt one term or the other and stick to it in this document.